swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition = None, age = None): # took category out and passed it in through super. Back to my original working way.
```

Comment:
Did you know that you can use [docstrings](https://www.python.org/dev/peps/pep-0257/) to comment a method.  That way it doesn't go too long on one line.